packages/fxa-payments-server/src/components/fields.tsx

```diff
@@ -116,7 +116,7 @@ export const Input = (props: InputProps) => {
   const onChange = useCallback(
     ev => {
       const { value } = ev.target;
-      validator.updateField({ name, value });
+      validator.updateField({ name, value, valid: true });
```

Comment:
flush invalidation when a user starts typing

Reply:
This also allows the zip code field to appear valid when filled with letters